app/repositories/RegistrationsRepository.scala

```diff
@@ -57,9 +57,9 @@ class DefaultRegistrationsRepository @Inject()(dateFormatter: DateFormatter,
   override def set(userAnswers: UserAnswers)(implicit hc: HeaderCarrier): Future[Boolean] = {
 
     val reference =
-        userAnswers.get(AgentInternalReferencePage).map {
-          reference => reference
-        }.orElse(None)
+      userAnswers.get(AgentInternalReferencePage).map {
```

Comment:
I know this isn't your code (in fact, it's probably something I did). Do we need the map here? I think it can just assign userAnswers.get here to the same effect.